steps/60-lab-docker-compose-solution/solution.sh

```diff
@@ -0,0 +1,8 @@
+#!/usr/bin/env bash
+
+docker build -t <dockerHubId>/my_flask:1.0 .
```

Comment:
Why not use docker-compose commands to build and run ?

Reply:
Oh this was a bad copy past, it was the same file as we use for the previous lab, it's fixed now